test/common/run-tests

```diff
@@ -67,83 +66,85 @@ def print_test(test, print_tap=True, retry_reason=""):
                 line = line[e.characters_written:]
                 time.sleep(0.1)
 
+    if retry_reason:
+        retry_reason = " " + retry_reason
+    if skip_reason:
+        skip_reason = " " + skip_reason
+
+
     if not print_tap:
-        print(retry_reason)
+        print((retry_reason or "") + (skip_reason or ""))
```

Comment:
Why the `or ""`? The default is already `""`, so this would only fail if you explicitly passed `None`? But, I see the code below in finish_test() sets them to `None`, so perhaps just make the default args None.

Reply:
right, because when I am parsing these reasons I default to None thus I can pass None directly. This is thinko. I'll init reasons to "" and then should be fine.